metricflow/model/data_warehouse_model_validator.py

```diff
@@ -341,25 +359,6 @@ def gen_measure_tasks(
                         error_message=f"Unable to query measure `{spec.element_name}` on data source `{data_source.name}` in data warehouse",
                     )
                 )
-
-            filter_elements_node = FilterElementsNode(parent_node=source_node, include_specs=data_source_specs)
-            tasks.append(
```

Comment:
The test failures are caused, at least in part, by removing this task. It does serve a purpose but @QMalcolm is a lot more familiar with it than I am so he'd know best how to get it fixed up.

Reply:
ok i realized what the differences here are. This union of all measure spec is the first task, then if it fails we call the individual querys from above

Reply:
there seems to be an issue tho with the validation